battles/tests/tests_forms.py

```diff
@@ -0,0 +1,100 @@
+from django.test import Client, TestCase
+
+from model_mommy import mommy
+
+from battles.forms import CreateBattleForm, SelectTrainerTeamForm
+
+
+class CreateBattleFormTest(TestCase):
+    def setUp(self):
+        self._user_password = '123456'
+        self.user = mommy.prepare('users.User', email='user@email.com')
+        self.user.set_password(self._user_password)
+        self.user.save()
+        self.auth_client = Client()
```

Comment:
Instead of instantiating the auth_client in every test setUp, you could have a `PokeBattleTestCase(TestCase)` which defines this on the setup and here, you would only do:
```python 
CreateBattleFormTest(PokeBattleTestCase):
    def setUp(self):
         super().setUp()
```

Reply:
Nice, I didn't know that before. Done! Thanks!